libvirt/cloudinit_def_test.go

```diff
@@ -41,16 +30,15 @@ func TestCloudInitCreateFiles(t *testing.T) {
 		t.Errorf("Unexpected error %v", err)
 	}
 	defer os.RemoveAll(dir)
-
-	for _, file := range []string{userData, metaData} {
```

Comment:
this variable were global shared between files and declared in other files. for the simple for loop is better to have them explicit as string